std/assembly/util/string.ts

```diff
@@ -603,6 +1039,55 @@ function pow10(n: i32): f64 {
   return load<f64>(Powers10.dataStart as usize + (n << alignof<f64>()));
 }
 
+function codePointBefore(buffer: usize, index: i32): i32 {
```

Comment:
Is this used exactly once? If so, perhaps `@inline`, since `isFinalSigma` is `@inline` too?

Reply:
Yeah, makes sense